src/test/scala/com/fulcrumgenomics/umi/UmiConsensusCallerTest.scala

```diff
@@ -57,4 +58,23 @@ class UmiConsensusCallerTest extends UnitSpec with OptionValues {
     warning shouldBe None
     error.value.indexOf("foo.bam") should be >= 0
   }
+
+  it should "header should output correct platform id" in {
```

Comment:
```suggestion
  "UmiConsensusCaller.outputHeader" should "output the correct platform id" in {
```

1. The `it` should be more specific, as it's testing a specific method, not the end-to-end tool.
2. `it should` will write "UmiConsensusCaller should ...." so no need to write "header should..." as then it would write "UmiConsensusCaller should header should output..." which makes no sense!